boards/wt32-eth02.json

```diff
@@ -0,0 +1,40 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_WT32_ETH01",
+      "-DCONFIG_FREERTOS_UNICORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "qio",
+    "mcu": "esp32",
+    "variant": "wt32-eth02"
+  },
+  "connectivity": [
+    "wifi",
+    "bluetooth",
+    "ethernet",
+    "can"
+  ],
+  "debug": {
+    "openocd_board": "esp32-solo-1.cfg"
+  },
+  "frameworks": [
+    "arduino",
```

Comment:
Not compatible with Arduino. The precompiled static Arduino libs are for dual core esp32 only.
Every Esp32solo1 does not work with Arduino

Reply:
ok, have removed arduino.